wordpress/wp-content/plugins/memberful-wp/src/content_filter.php

```diff
@@ -13,6 +13,9 @@ function memberful_wp_protect_content( $content ) {
     // Disable Beaver Builder
     remove_action( "the_content", "FLBuilder::render_content" );
 
+    // Remove Elementor action hook
+    remove_action("elementor/frontend/the_content", "memberful_wp_protect_content");
```

Comment:
@northerner OK, I had to check Elementor's source code, but it looks that this is OK (it's still awful callback hell, but what can we do 🤷‍♂️ 😅).